### PR TITLE
Don fixed HelpWindow CSS, UG

### PR DESCRIPTION
**Pull request by Donavanty**
1. Fixed the CSS of HelpWindow
2. Touched-up the whole UG, do take a quick look

### CODE REVIEW

File: src/main/resources/view/HelpWindow.fxml [hunk 1]
```diff
@@ -16,16 +17,16 @@
   </icons>
   <scene>
     <Scene fill="BLACK">
-         <VBox prefHeight="380.0" prefWidth="548.0" style="-fx-background-color: #383838;">
+         <VBox prefHeight="380.0" prefWidth="548.0" styleClass="background">
             <children>
-               <TextArea fx:id="helpMessageField" editable="false" scaleShape="false" style="-fx-control-inner-background: derive(#1d1d1d, 30%); -fx-text-fill: white; -fx-border-color: transparent; -fx-background-insets: 0; -fx-text-box-border: transparent; -fx-focus-color: #383838; -fx-background-color: transparent; " wrapText="true" VBox.vgrow="ALWAYS" />
+               <TextArea fx:id="helpMessageField" editable="false" scaleShape="false" styleClass="help_text_area" wrapText="true" VBox.vgrow="ALWAYS" />
```
Comment (Donavanty) [inline src/main/resources/view/HelpWindow.fxml:22]: Removed internal styling in the FXML file, put into CSS instead

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -41,8 +41,8 @@
         </StackPane>
 
 
-        <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" minHeight="200" styleClass="pane-with-border"
-                   prefHeight="100" maxHeight="100">
+        <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" minHeight="150" styleClass="pane-with-border"
+                   prefHeight="150">
```
Comment (Donavanty) [inline src/main/resources/view/MainWindow.fxml:45]: Reduced the window size (height) of the resultDisplayPlaceholder from 200 to 150, felt that it looked better

**Comment by Donavanty** [comment]
Just touched-up the UG, have a look at your sections :)

**Review by marcusleeeugene — APPROVED** [review]
LGTM!

**Pull request merged**